src/main/java/calculator/model/Operator.java

```diff
@@ -0,0 +1,35 @@
+package calculator.model;
+
+import calculator.exception.InvalidOperatorException;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+
+    ADD("+", Number::add),
+    SUBTRACT("-", Number::subtract),
+    MULTIPLY("*", Number::multiply),
+    DIVIDE("/", Number::divide);
+
+    private final String value;
+
+    private final BiFunction<Number, Number, Number> calculator;
```

Comment:
```suggestion
    private final BinaryOperator<Number> calculator;
```

매개변수, 반환값이 같기 때문에 BinaryOperator로도 사용해 볼 수 있겠네요 😄